Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -193,19 +193,39 @@ private string GetFixedTableHeaderMaxHeight()
             return FixedHeaderDataGridMaxHeight;
     }
 
+    /// <summary>
+    /// Returns a list of all the columns that are currently associated with this datagrid.
+    /// </summary>
+    /// <returns>A read-only list of all columns</returns>
+    public IReadOnlyList<DataGridColumn<TItem>> GetColumns() => Columns.AsReadOnly();
+
+    /// <summary>
+    /// Returns a list of all columns currently used to sort this datagrid's data.
+    /// </summary>
+    /// <returns>A read-only list of all sort columns, or an empty list.</returns>
+    public IReadOnlyList<DataGridColumn<TItem>> GetSortColumns() => SortByColumns.AsReadOnly();

```

Comment:
Should we just make the existing SortByColumns API public instead of adding a new `GetSortColumns()` API?

Also, rename it to `GetSortByColumns`?

Reply:
Same answer as for `GetColumns`. I'm good with the renaming to be consistent with the underlying collection.

Reply:
Renamed. Please resolve this if you're ok with the API.